charts/linkerd2/README.md

```diff
@@ -102,6 +100,8 @@ their default values.
 | `global.clusterDomain`                      | Kubernetes DNS Domain name to use                                                                                                                                                     | `cluster.local`                      |
 | `global.cniEnabled`                         | Omit the NET_ADMIN capability in the PSP and the proxy-init container when injecting the proxy; requires the linkerd-cni plugin to already be installed                               | `false`                              |
 | `global.controllerComponentLabel`           | Control plane label. Do not edit                                                                                                                                                      | `linkerd.io/control-plane-component` |
+| `global.controllerImageVersion`             | Tag for the controller container docker image                                                                                                                                         | latest version                       |
```

Comment:
controllerImageVersion and global.controllerImageVersion both exist.  I don't understand what the difference between them is or why they both exist though.

Reply:
@adleong where do you see `controllerImageVersion` outside the `global` context?

Reply:
https://github.com/linkerd/linkerd2/blob/main/pkg/charts/linkerd2/values.go#L26

Reply:
Ok I think the one without `global.` is some leftover from a previous refactoring, as it's not used anywhere in the linkerd2 chart templates. If so, we should remove it from values.go, options.go and some other places.